zookeeper/src/test/java/com/linecorp/armeria/client/zookeeper/TestBase.java

```diff
@@ -49,7 +49,9 @@
     @BeforeClass
     public static void start() {
         try {
-            zkInstance = ZKFactory.apply().withRootDir(ROOT_FOLDER.newFolder("zookeeper")).create();
+            if (zkInstance == null) {
```

Comment:
Yeah - that's what the code is changed too, but I was asking why this is needed considering we're stopping the server. 

I poked around in the tests and noticed that many connections are made without closing them. Closing them removed the spam! So maybe #433 is a cleaner way than being forced to reuse zkInstance.